# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xsupplicant/xsupplicant-1.2.8-linux-headers.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

http://bugs.gentoo.org/attachment.cgi?id=141784&action=view
diff -Nur ./src/cardif/linux/cardif_atmel_driver.c.orig ./src/cardif/linux/cardif_atmel_driver.c
--- ./src/cardif/linux/cardif_atmel_driver.c.orig	2006-10-08 05:42:53.000000000 +0200
+++ ./src/cardif/linux/cardif_atmel_driver.c	2008-01-25 15:45:56.000000000 +0100
@@ -103,7 +103,6 @@
 #include <sys/ioctl.h>
 #include <linux/types.h>
 #include <linux/socket.h>
-#include <linux/compiler.h>
 #include <iwlib.h>
 #include <stdlib.h>
 #include <string.h>
diff -Nur ./src/cardif/linux/cardif_linux.c.orig ./src/cardif/linux/cardif_linux.c
--- ./src/cardif/linux/cardif_linux.c.orig	2006-10-08 05:42:53.000000000 +0200
+++ ./src/cardif/linux/cardif_linux.c	2008-01-25 15:46:09.000000000 +0100
@@ -174,7 +174,6 @@
 #include <netinet/in.h>
 #include <linux/types.h>
 #include <linux/socket.h>
-#include <linux/compiler.h>
 #include <iwlib.h>
 #include <linux/if_packet.h>
 #include <stdio.h>
diff -Nur ./src/cardif/linux/cardif_linux_rtnetlink.c.orig ./src/cardif/linux/cardif_linux_rtnetlink.c
--- ./src/cardif/linux/cardif_linux_rtnetlink.c.orig	2006-10-08 05:42:53.000000000 +0200
+++ ./src/cardif/linux/cardif_linux_rtnetlink.c	2008-01-25 15:46:21.000000000 +0100
@@ -203,7 +203,6 @@
 
 #include <unistd.h>
 #include <sys/socket.h>
-#include <linux/compiler.h>
 #include <iwlib.h>
 #include <sys/ioctl.h>
 #include "netlink.h"
diff -Nur ./src/cardif/linux/cardif_linux_wext.c.orig ./src/cardif/linux/cardif_linux_wext.c
--- ./src/cardif/linux/cardif_linux_wext.c.orig	2006-10-08 05:42:53.000000000 +0200
+++ ./src/cardif/linux/cardif_linux_wext.c	2008-01-25 15:46:31.000000000 +0100
@@ -203,7 +203,6 @@
 #include <netinet/in.h>
 #include <linux/types.h>
 #include <linux/socket.h>
-#include <linux/compiler.h>
 #include <linux/wireless.h>
 #include <iwlib.h>
 #include <linux/if_packet.h>
